### DISCUSSION TITLE
Sessions are not recored on k8s

### DISCUSSION THREAD

**Discussion by Dev-OrwaSalim** [gravitational/teleport#49981] [Q&A]
Hi
I have deployed Teleport community version v17.0.3 through Helm on EKS cluster, everything woking fine but there issue that sessions are not recorded when a user exec a pod through teleport!

I found in teleport-auth pod's logs that session is uploaded to s3 and i found it as tar file on s3 but that tar file is empty!
and there no session records on teleport UI

Previous I have teleport installed on EC2 server and session was recording successfully, now when i move it to k8s it doesn't record the session

Can you please help me with this
Thank you

**Comment by webvictim** [comment]
You mentioned S3. Can you share your Helm chart values here? (redact any tokens or sensitive information but please leave as much intact as you can)
